TundraTD/Assets/Scripts/Level/LevelCornerman.cs

```diff
@@ -52,7 +55,13 @@ private IEnumerator StartWavesLoop()
                     mobPortal.EquipNextWave();
                     yield return StartMobSpawning(mobPortal);
                 }
-                yield return new WaitUntil(() => mobPortals.Sum(x => x.MobsLeftThisWave) == 0);
+
+                foreach (var portal in mobPortals)
+                {
+                    portal.MobPool.AreAllMobDead();
+                }
```

Comment:
I don't understand this loop. Below you check if all pools are full, but why is this logic here?